tests/README.md

```diff
@@ -0,0 +1,178 @@
+# Crypto tests
+
+## Running crypto Perl tests
+
+Crypto tests require the Math::GMP Perl library, make sure it is installed on you system before running the tests.
+
+Installing dependencies (using cpan):
+
+```
+cpan
+cpan> install Math::BigInt::GMP
+cpan> install Digest::Keccak
+```
+
+Running tests:
+
+```
+TESTPATH=/path/to/monero/tests
+cd $TESTPATH
+perl -I $TESTPATH cryptotest.pl
+```
+
+Important: must include test path for perl to import cryptolib.pl
+
+## Writing new crypto tests
+
+[TODO]
+
+# Core tests
+
+## Running core tests
+
+Monero uses the Google C++ Testing Framework (`gtest`) to write unit, integration and functional tests for core and other features of the project.
+`gtest` runs on top of cmake, and you can run all tests by:
```

Comment:
AFAIK, core tests don't use gtest. Only unit tests do. And libwallet tests, but these are weird.

Reply:
Definitely my mistake, was wondering if gtest was used for the entire test suite (not super familiar with gtest). I'll edit that part to only reference unit tests, and give specific entries for integration and functional tests.